### PR TITLE
Fix: Drop plasteel when deconstructing reinforced walls

### PR DESCRIPTION
**Pull request by Carbonhell**
## About the PR 

**Changelog**

:cl: Carbonhell
- tweak: Deconstructing reinforced walls now leads to the creation of a reinforced girder instead of a normal one, and it drops 2 plasteel sheets.
- tweak: Deconstructing reinforced girders now only requires cutting for 2 seconds and drops 2 plasteel sheets.

### CODE REVIEW

**Review by Just-a-Unity-Dev — APPROVED** [review]

**Comment by Just-a-Unity-Dev** [comment]
based first PR

File: Resources/Prototypes/Recipes/Construction/Graphs/structures/girder.yml [hunk 1]
```diff
@@ -250,6 +250,10 @@
       entity: WallReinforced
       edges:
         - to: girder
+          completed:
+            - !type:SpawnPrototype
+              prototype: SheetPlasteel1
+              amount: 4
```
Comment (mirrorcult) [inline Resources/Prototypes/Recipes/Construction/Graphs/structures/girder.yml:256]: seems like this should honestly be an edge from `reinforcedWall` -> `reinforcedGirder` dropping 2 plasteel, then the `reinforcedGirder` -> `girder` edge can drop the other 2 plasteel

Comment (Carbonhell) [reply Resources/Prototypes/Recipes/Construction/Graphs/structures/girder.yml:256]: Yeah that makes more sense, I will do the change as soon as possible, thank you!

Comment (Carbonhell) [reply Resources/Prototypes/Recipes/Construction/Graphs/structures/girder.yml:256]: Actually, there is an issue. The reinforced girder requires a welding step to deconstruct it in a normal girder. That means this change would increase the welding steps from 2 to 3 to completely take down a reinforced wall. Isn't it a bit too much? I think a possible change would be to swap the Welding step with a Cutting step on the reinforced girder -> girder graph, but I don't know if there's anything balance-wise to discuss regarding this.

**Comment by Carbonhell** [comment]
As said on Discord, I've changed the reinforced wall decon graph to lead to the creation of a reinforced girder (now promoted to an entity) rather than a normal one. The full deconstruction still drops 4 plasteel sheets.

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**